command.go

```diff
@@ -62,6 +62,7 @@ type commandContext struct {
 
 	selected      []mat.Vec3
 	selectedStack [][]mat.Vec3
+	selectMask    []uint32
```

Comment:
Move point select mask buffer to here to avoid passing it to every functions of `commandContext` and `consoleCommands`.